src/Sarif.Multitool.UnitTests/TransformCommandTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Multitool;
+using Moq;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Sarif.Multitool.UnitTests
+{
+    public class TransformCommandTests
+    {
+        // Regression test for Issue #1064, "ValidatorCommand fails when target file name contains a space"
```

Comment:
>space [](start = 101, length = 5)

But the target file name below, `c:\logs\mylog.sarif`, does not contain a space.

Reply:
comment left from copying another test class to use as basis for this one.

---
In reply to: [229507635](https://github.com/Microsoft/sarif-sdk/pull/1125#discussion_r229507635) [](ancestors = 229507635)